### PR TITLE
Feature/fixing UI

### PR DESCRIPTION
**Pull request by an-desco**

### CODE REVIEW

File: client/src/components/entities/lights/things/ColorTemperaturePicker.svelte [hunk 1]
```diff
@@ -15,12 +15,14 @@
...
-<input
-    type="range"
-    min="153"
-    max="500"
-    bind:value
-    on:change={tempColorChangeHandler}
-/>
-{value.toFixed(0)}°?
+<div class="w-inherit">
+    <p>Color temperature : {value.toFixed(0)}°?</p>
```
Comment (souziel) [inline client/src/components/entities/lights/things/ColorTemperaturePicker.svelte]: Can you use i18n ?

File: client/src/components/entities/switchs/Switch.svelte [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+            <h1 class="text-4xl text-center py-6">{entity.name}</h1>
+        </div>
+
+        <div id="content" class="flex flex-row justify-center">
+            <div
+                id="info"
+                class="relative grid grid-cols-12 gap-4 py-4 uppercase bg-esidomlight w-full justify-center md:w-2/3 md:max-w-xl"
+            >
+                <div class="col-span-full text-center font-semibold text-xl">
+                    Informations
```
Comment (souziel) [inline client/src/components/entities/switchs/Switch.svelte:42]: Can you use i18n ?

File: client/src/components/entities/lights/things/ColorTemperaturePicker.svelte (hunk 1 above)
Comment (an-desco) [reply client/src/components/entities/lights/things/ColorTemperaturePicker.svelte]: solved

**Review by asoran — APPROVED** [review]
🐲

**Review by guyts96 — APPROVED** [review]

**Pull request merged**